src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -729,6 +729,8 @@ static Option firstVariadicArg(int index) {
          * }
          * }
          *
+         * @apiNote This linker option can not be combined with {@link #isTrivial}.
+         *
```

Comment:
I'm not sure if we should a similar API note on `isTrivial`. Just having one here seems enough?

Reply:
I think it's enough to have it here